### DISCUSSION TITLE
打开“插件市场”页面时报错

### DISCUSSION THREAD

**Discussion by Northm0in** [bs-community/blessing-skin-server#459] [Q&A]
如下
>**严重错误**
>Undefined array key "packages"
>▼ 详情
>[1] app/Http/Controllers/MarketController.php#L91
>[2] app/Http/Controllers/MarketController.php#L95
>[3] app/Http/Controllers/MarketController.php#L20
>[4] app/Http/Middleware/CheckRole.php#L22
>[5] app/Http/Middleware/FireUserAuthenticated.php#L15
>[6] app/Http/Middleware/EnsureEmailFilled.php#L17
>[7] app/Http/Middleware/RejectBannedUser.php#L23
>[8] app/Http/Middleware/RedirectToSetup.php#L28
>[9] app/Http/Middleware/EnforceEverGreen.php#L23
>[10] app/Http/Middleware/DetectLanguagePrefer.php#L30
>[11] app/Http/Middleware/ConvertEmptyStringsToNull.php#L23

**Comment by sh1roko516** [comment]
在.env里面加下面的内容：
PLUGINS_REGISTRY=https://git.qvq.network/bs-community/plugins-dist/-/raw/master/registry_{lang}.json

**Reply by Yukino12333** [reply]
依旧不行
严重错误
×
cURL error 35: Peer reports incompatible or unsupported protocol version. (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://git.qvq.network/bs-community/plugins-dist/-/raw/master/registry_zh_CN.json
详细信息
[1] app/Http/Controllers/MarketController.php#L88
[2] app/Http/Controllers/MarketController.php#L95
[3] app/Http/Controllers/MarketController.php#L20
[4] app/Http/Middleware/CheckRole.php#L22
[5] app/Http/Middleware/FireUserAuthenticated.php#L15
[6] app/Http/Middleware/EnsureEmailFilled.php#L17
[7] app/Http/Middleware/RejectBannedUser.php#L23
[8] app/Http/Middleware/RedirectToSetup.php#L28
[9] app/Http/Middleware/EnforceEverGreen.php#L23
[10] app/Http/Middleware/DetectLanguagePrefer.php#L30
[11] app/Http/Middleware/ConvertEmptyStringsToNull

**Reply by Big-Cake-jpg** [reply]
> 依旧不行
> 严重错误
> ×
> cURL error 35: Peer reports incompatible or unsupported protocol version. (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://git.qvq.network/bs-community/plugins-dist/-/raw/master/registry_zh_CN.json
> 详细信息
> [1] app/Http/Controllers/MarketController.php#L88
> [2] app/Http/Controllers/MarketController.php#L95
> [3] app/Http/Controllers/MarketController.php#L20
> [4] app/Http/Middleware/CheckRole.php#L22
> [5] app/Http/Middleware/FireUserAuthenticated.php#L15
> [6] app/Http/Middleware/EnsureEmailFilled.php#L17
> [7] app/Http/Middleware/RejectBannedUser.php#L23
> [8] app/Http/Middleware/RedirectToSetup.php#L28
> [9] app/Http/Middleware/EnforceEverGreen.php#L23
> [10] app/Http/Middleware/DetectLanguagePrefer.php#L30
> [11] app/Http/Middleware/ConvertEmptyStringsToNull

求你了，看看 Telegram 频道怎么说的吧……

新的插件源只支持 TLS 1.3，你需要保证 PHP 使用的 OpenSSL 版本在 1.1.1 以上并开启 TLS 1.3 支持
